sci-astronomy/stellarium/stellarium-24.1.recipe

```diff
@@ -3,12 +3,11 @@ DESCRIPTION="Stellarium is a free open source planetarium for your computer. \
 It shows a realistic sky in 3D, just like what you see with the naked eye, \
 binoculars or a telescope."
 HOMEPAGE="https://stellarium.org"
-COPYRIGHT="2001-2022 Fabien Chéreau"
+COPYRIGHT="2001-2024 Fabien Chéreau"
 LICENSE="GNU GPL v2"
-REVISION="1"
-SOURCE_URI="https://github.com/Stellarium/stellarium/archive/refs/tags/v$portVersion.tar.gz"
-CHECKSUM_SHA256="841dfbd27fa6ea97c4ce22424b12a9e73ad70d61af892734ce2f67d7681ec76d"
-PATCHES="stellarium-$portVersion.patchset"
+REVISION="2"
```

Comment:
Revision should stay on 1, on newer version they are "reset" to 1 where needed, on a revision bump the number is going up.